src/backends/perf_pt/mod.rs

```diff
@@ -532,9 +532,9 @@ mod tests {
     // Given a trace, use ptxed to get a vector of block start vaddrs.
     fn get_expected_blocks(trace: &Box<Trace>) -> Vec<Block> {
         // Write the trace out to a temp file so ptxed can decode it.
-        let mut fh = NamedTempFile::new().unwrap();
```

Comment:
Why the renaming of `fh` to `tmpf`? If we leave it as `fh` the change touches fewer lines (OK, only one, but still).

Reply:
Because it's no longer a file handle...

Reply:
> Because it's no longer a file handle

Well, it was a handle on a temporary file before, and that's what it still is.

I don't care greatly, but I think it's good practice to touch as few lines as possible unless there's a good reason.

Up to you...

Reply:
I feel like you are splitting hairs, but as I said, it's up to you. So are you ready to merge?

Reply:
Yep!